app/src/main/java/org/mozilla/fenix/settings/LearnMoreSwitchPreference.kt

```diff
@@ -28,11 +28,17 @@ abstract class LearnMoreSwitchPreference(context: Context, attrs: AttributeSet?)
 
     override fun onBindViewHolder(holder: PreferenceViewHolder) {
         super.onBindViewHolder(holder)
-
+        holder.itemView.setOnClickListener { }
```

Comment:
Here it should work when the user presses on the switch or on the sharepref item. When you press on the item, it no longer works and only works for the switch.I think it should work in both cases, like in Focus.

Reply:
In Fenix we have some settings that only work when clicking the switch for example the HTTPS-only mode. As this widget is a generic way to not re-do the work for the learn more link, I think is a bearable trade-off :)

Reply:
So if you what the item to behave like Https-only mode you should put         holder.itemView.isClickable = false
instead of         holder.itemView.setOnClickListener { } because now the item is clickable but it doesn't do anything.